tests/unit/routes/rentals/show-test.js

```diff
@@ -1,11 +1,14 @@
 import { moduleFor, test } from 'ember-qunit';
 
-moduleFor('route:rentals/show', 'Unit | Route | rentals/show', {
-  // Specify the other units that are required for this test.
-  // needs: ['controller:foo']
-});
+moduleFor('route:rentals/show', 'Unit | Route | rentals/show', {});
 
-test('it exists', function(assert) {
-  let route = this.subject();
-  assert.ok(route);
+test('queryRecord hits the API with the correct credentials', function(assert) {
```

Comment:
I've been starting my tests with "should", so that when run they end up looking like:
`Unit | Route | rentals/show should query for selected rental by slug`